tikvrpc/tikvrpc.go

```diff
@@ -731,104 +743,31 @@ type MPPStreamResponse struct {
 	Lease
 }
 
+//go:generate bash gen.sh
+
 // AttachContext sets the request context to the request,
 // return false if encounter unknown request type.
 // Parameter `rpcCtx` use `kvrpcpb.Context` instead of `*kvrpcpb.Context` to avoid concurrent modification by shallow copy.
```

Comment:
The comments need to be updated, we should also add a WARN here like `do not edit Request outisde of this method, otherwise there could be data race`

Reply:
Added to `Request.Req`.